scripts/Dockerfile.ecosystem

```diff
@@ -8,7 +8,8 @@
 #
 # From the project root:
 # ```
-# cargo build
+# rustup target add x86_64-unknown-linux-musl
```

Comment:
Nice... do we need to disable jemalloc because I run into:

error: failed to run custom build command for `tikv-jemalloc-sys v0.5.3+5.3.0-patched`

Reply:
huh what os are you building on?

Reply:
Manjaro (arch) x86.... 

It seems that some files are missing

```
  --- stderr
  running "musl-gcc" "-O3" "-ffunction-sections" "-fdata-sections" "-fPIC" "-m64" "-I" "include" "-Wall" "-Wextra" "-pedantic" "-pedantic-errors" "-Wall" "-Wextra" "-Wcast-align" "-Wcast-qual" "-Wconversion" "-Wenum-compare" "-Wfloat-equal" "-Wformat=2" "-Winline" "-Winvalid-pch" "-Wmissing-field-initializers" "-Wmissing-include-dirs" "-Wredundant-decls" "-Wshadow" "-Wsign-compare" "-Wsign-conversion" "-Wundef" "-Wuninitialized" "-Wwrite-strings" "-fno-strict-aliasing" "-fvisibility=hidden" "-fstack-protector" "-g3" "-U_FORTIFY_SOURCE" "-DNDEBUG" "-c" "-o/home/micha/astral/ruff/target/x86_64-unknown-linux-musl/release/build/ring-6202016dbf886143/out/aesni-x86_64-elf.o" "/home/micha/.cargo/registry/src/github.com-1ecc6299db9ec823/ring-0.16.20/pregenerated/aesni-x86_64-elf.S"
  thread 'main' panicked at 'failed to execute ["musl-gcc" "-O3" "-ffunction-sections" "-fdata-sections" "-fPIC" "-m64" "-I" "include" "-Wall" "-Wextra" "-pedantic" "-pedantic-errors" "-Wall" "-Wextra" "-Wcast-align" "-Wcast-qual" "-Wconversion" "-Wenum-compare" "-Wfloat-equal" "-Wformat=2" "-Winline" "-Winvalid-pch" "-Wmissing-field-initializers" "-Wmissing-include-dirs" "-Wredundant-decls" "-Wshadow" "-Wsign-compare" "-Wsign-conversion" "-Wundef" "-Wuninitialized" "-Wwrite-strings" "-fno-strict-aliasing" "-fvisibility=hidden" "-fstack-protector" "-g3" "-U_FORTIFY_SOURCE" "-DNDEBUG" "-c" "-o/home/micha/astral/ruff/target/x86_64-unknown-linux-musl/release/build/ring-6202016dbf886143/out/aesni-x86_64-elf.o" "/home/micha/.cargo/registry/src/github.com-1ecc6299db9ec823/ring-0.16.20/pregenerated/aesni-x86_64-elf.S"]: No such file or directory (os error 2)', /home/micha/.cargo/registry/src/github.com-1ecc6299db9ec823/ring-0.16.20/build.rs:653:9
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

```

Running `cargo clean` nor building with `--release` helped

Reply:
Solution:

I had to install `musl` (`yay musl`). For debian `apt-get install musl-tools`.

Reply:
how does this compare to the experience with zig? i'm trying to figure out which one to recommend

Reply:
It's very similar. With Zig, I had to install `zig` and I had to install `musl` with this approach.

I'm a complete docker noob but is there an image (even if slightly larger in size) that uses a more recent `glibc` version to avoid cross-compiling?

Reply:
thanks, added it to the documentation